tests/integration/ceph_base_block_test.go

```diff
@@ -45,6 +46,231 @@ func checkSkipCSITest(t *testing.T, k8sh *utils.K8sHelper) {
 	}
 }
 
+func checkSkipSnapshotTest(t *testing.T, k8sh *utils.K8sHelper) {
+	minVersion := "v1.17.0"
+	if !k8sh.VersionAtLeast(minVersion) {
+		logger.Infof("Skipping snapshot tests as kubernetes version is less than %q for the CSI driver", minVersion)
+		t.Skip()
```

Comment:
If you call `Skip()`, the entire test will be skipped after this. The snaphsot and clone tests are not dedicated just to snapshots or clones, right? Since they are just embedded in another integration test, you'll want to just return false and skip with your own code instead of `Skip()`